### PR TITLE
Use brunch auto-require

### PR DESCRIPTION
**Pull request by gjaldon**
Closes https://github.com/phoenixframework/phoenix/issues/1067

@josevalim hope you don't mind I went ahead and created a PR for above issue. :)

Heads up - this can only be merged once brunch 1.8.4 has been released https://github.com/brunch/brunch/pull/1001.

### CODE REVIEW

File: installer/templates/static/brunch/brunch-config.js [hunk 1]
```diff
@@ -51,6 +51,13 @@ exports.config = {
       ignore: [/web\/static\/vendor/]
     }
   },
+
+  modules: {
+    autoRequire: {
+      'js/app.js': ['app']
```
Comment (josevalim) [inline installer/templates/static/brunch/brunch-config.js]: Have you tried this out? Because I think the require should be `web/static/js/app`.

Comment (gjaldon) [inline installer/templates/static/brunch/brunch-config.js]: Changed it and tried it out with the ff brunch_config.js and app.js https://gist.github.com/gjaldon/22190474a0579513fb99. It doesn't seem to load app.js. This is on a new Phoenix app with brunch master. Am I doing it right?

File: installer/templates/static/brunch/brunch-config.js [hunk 2]
```diff
@@ -51,6 +51,13 @@ exports.config = {
       ignore: [/web\/static\/vendor/]
     }
   },
+
+  modules: {
+    autoRequire: {
+      'web/static/js/app.js': ['app']
```
Comment (josevalim) [inline installer/templates/static/brunch/brunch-config.js]: The left side was correct, you had to change the right side to mirror what you removed from the layout.

Comment (gjaldon) [inline installer/templates/static/brunch/brunch-config.js]: Thanks a lot! :) This is fixed now.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Comment by paulmillr** [comment]
Push it now.

**Comment by paulmillr** [comment]
:smirk_cat:

**Comment by chrismccord** [comment]
Thanks folks! This is such a nice solution vs the alternatives

**Pull request merged**